### PR TITLE
Updated the readme doc

### PR DESCRIPTION
**Pull request by pintu-eGov**

### CODE REVIEW

File: domain-services/ifix-department-entity-service/README.md [hunk 1]
```diff
@@ -27,5 +27,20 @@ Children list contains department entity id list, which makes current department
...
+             
+             1. Find the department entity which hierarchy level is less than one by current department entity's hierarchy level.
+              
+              db.departmentEntity.find({"departmentId" : "<current_department_entity's_department_id>","hierarchyLevel": <Current_department_entity_hierarchy_level- 1>});
+              
+              2. Append the resulted department entity id in the Current department entity's children list 
+             
+             db.departmentEntity.update({"_id": "<current_department_entity_id>"},{$set:
+             {"children.0": "<Resulted_department_entity_id_1>",
+             "children.1": "<Resulted_department_entity_id_2>"}
```
Comment (rushang7-eGov) [inline domain-services/ifix-department-entity-service/README.md:45]: Write that, you need to append to the end of the array. So first find the length of the current array and then set it as
"children.n": "NewDepartmentEntityId" (where n is the length of the array.

File: domain-services/ifix-department-entity-service/README.md [hunk 2]
```diff
@@ -27,5 +27,20 @@ Children list contains department entity id list, which makes current department
...
+_**Note**: 
+1. The root department hierarchy will have the label "Department", and the root department entity will be the
 department itself._
+2. When we have to update the existing children list of Department Entity then update the existing children list using mongodb command like below:
+
+   **Steps**: 
+             
+             1. Find the department entity which hierarchy level is less than one by current department entity's hierarchy level.
+              
+              db.departmentEntity.find({"departmentId" : "<current_department_entity's_department_id>","hierarchyLevel": <Current_department_entity_hierarchy_level- 1>});
```
Comment (rushang7-eGov) [inline domain-services/ifix-department-entity-service/README.md:39]: Write: we need to find the department entity parent where the new children need to be added. Do search by name and hierarchy level, and not just hierarchy level.

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

File: domain-services/ifix-department-entity-service/README.md (hunk 1 above)
Comment (pintu-eGov) [reply domain-services/ifix-department-entity-service/README.md]: done

File: domain-services/ifix-department-entity-service/README.md (hunk 2 above)
Comment (pintu-eGov) [reply domain-services/ifix-department-entity-service/README.md]: done

**Review by rushang7-eGov — APPROVED** [review]

**Pull request merged**